### ISSUE TITLE
Proper naming for the gridlines 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#120]
During the interviews for RaP it became clear that saying grid lines in the description was not the proper way to be referring to the lines so in todays meeting with @emily-phet and @terracoda we wanted to get your input on what we should be calling the lines.

**Non-Numbered lines**
![gridlines](https://user-images.githubusercontent.com/48224214/89681056-0600a680-d8a9-11ea-88a2-bf0acf152300.PNG)

**Numbered Lines** 
![numbered gridlines](https://user-images.githubusercontent.com/48224214/89681094-19137680-d8a9-11ea-8c3c-dd09860c0ada.PNG)

**Comment by emily-phet** [comment]
@amanda-phet 
It sounded like learners using screen readers were (correctly!) envisioning "boxes" formed by having horizontal and vertical lines when they heard "grid". 

Some ideas we had included:
- lines (maybe not specific enough...?)
- horizontal lines (though probably not great for younger learners)

If you have any suggestions, or know of any terms we should definitely avoid, please let us know.

**Comment by amanda-phet** [comment]
Other ideas... these lines function very similar to tick marks on a number line, so I think you could try "tick marks" and "numbered tick marks". I like that idea a lot, but if you want to limit words, perhaps "units" and "numbered units". However, units can be a loaded term, so I think tick marks really fits well here. If you don't love it, I can keep brainstorming!

**Comment by terracoda** [comment]
I like "tick marks" and "numbered tick marks", and I was also wondering if "guides" or "guide lines", or "numbered guides" and "numbered guide lines" might be appropriate.

Again, I like "tick marks", so if everyone likes that, let's go with that.

**Comment by terracoda** [comment]
Maybe the options could be referred to as "Guide Options" and then talk about "tick marks" as the guides in the help text and and the responses.

**Comment by terracoda** [comment]
For example, would this work?:

H3: Guide Options
P: Choose how you see or hear guided exploration. 
- Tick Marks
- Numbered Tick Marks
- No Tick Marks

**Comment by terracoda** [comment]
I have two related questions for @emily-phet and @amanda-phet:

1. Can we start with the non-numbered tick marks experience?
2. Would it be an okay option to use ordinals to describe the non-numbered tick marks? Or is it better to just describe the numbers in both cases?

**Comment by terracoda** [comment]
For example:
Currently, {{at}} the challenge {{1}} ratio with
- Left hand {{on second tick mark}} 
- Right hand {{on fourth tick mark}}

**Comment by terracoda** [comment]
Compared with numbered dynamic description which would be:
Currently, {{at}} the challenge {{1}} ratio with
- Left hand {{on tick mark 2}} 
- Right hand {{on tick mark 4}}

**Comment by amanda-phet** [comment]
>Can we start with the non-numbered tick marks experience?

This seems like a question for the design team, especially Brett who has been doing interviews. I assumed it was important to start with no tick marks (especially since it seems likely that students eventually spend more time with numbered tick marks on). 

>Would it be an okay option to use ordinals to describe the non-numbered tick marks? Or is it better to just describe the numbers in both cases?

Is "ordinals" the example of "on second tick mark" and "on fourth tick mark"? If so, I think that sounds fine.

**Comment by terracoda** [comment]
Thanks @amanda-phet, I only meant to ask you the ordinals questions :-)

**Comment by emily-phet** [comment]
@terracoda 
Regarding your question "Can we start with the non-numbered tick marks experience?". 
If you mean can the focus order start there, or the default startup of the sim start there, unfortunately no. The overall intended experience is for learners to explore non-numerically to start. Things like gridlines/tick marks and numbers are using mathematical representations. Dor's work on this topic is about starting pre-mathematical notation, and then layering in the mathematical notation piece by piece - first with gridlines, then with gridlines + numbers. So the flow of the sim must stay that way - with the "no gridlines" first. 

My suggestion last week to focus on quantitative descriptions was *not* to mean that there would never be qualitative descriptions (to align with the initial state of the sim on startup, with the "no gridlines" option selected), but to instead move forward with a doable option (quantitative) given our timeline to first publication. My thought was that the quantitative description would be provided in the "no gridlines" option until later when a qualitative solution was found. We should not try to re-scaffold the sim to make gridlines found sooner.

**Comment by terracoda** [comment]
@emily-phet, thanks for the clarification! This makes total sense, and this is why I double checked.
I don't think it makes sense either to change the order, but as you say for non-visual access we may not find that perfect non-visual qualitative experience before first publication.

**Comment by emily-phet** [comment]
I think we have a strategy to move forward with. Closing this issue and will open new ones specific to what we would like implemented.
